bin/server/cmd/component.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"net/url"
+	"os"
+
+	"github.com/bytebase/bytebase/common"
+	dbdriver "github.com/bytebase/bytebase/plugin/db"
+	"github.com/bytebase/bytebase/store"
+	"go.uber.org/zap"
+)
+
+func (m *metadataDB) connect() (*store.DB, error) {
+	if useEmbedDB() {
+		if err := m.pgInstance.Start(m.profile.datastorePort, os.Stderr, os.Stderr); err != nil {
```

Comment:
m.pgInstance only exists in the omni build. If we build with slim tag, this won't compile.